docs/faq.md

```diff
@@ -54,21 +54,20 @@ Griptape Nodes will automatically check if it needs to update every time it runs
 
 If you would like to _manually_ update, you can always use either of these commands:
 
-```griptape-nodes update```
+`griptape-nodes update`
```

Comment:
ah, I think the trick is to keep newlines..

![image](https://github.com/user-attachments/assets/29ebe26b-8740-43b2-8940-81c3e8ecf3f5)